src/cascadia/TerminalApp/Tab.cpp

```diff
@@ -64,15 +84,71 @@ void Tab::SetFocused(bool focused)
     }
 }
 
-GUID Tab::GetProfile() const noexcept
+// Method Description:
+// - Returns nullopt if no children of this tab were the last control to be
+//   focused, or the GUID of the profile of the last control to be focused (if
+//   there was one).
+// Return Value:
+// - nullopt if no children of this tab were the last control to be
+//   focused, else the GUID of the profile of the last control to be focused
+std::optional<GUID> Tab::GetLastFocusedProfile() const noexcept

```

Comment:
verbage: `Last` seems unnecessary? a tab has a focused thing; "last" implies that it was the one before the current one

Reply:
I was trying to mentally identify that the control might not _currently_ be focused, if the tab isn't. But maybe we'll keep that to the `Pane`, the tab's method's name could definitely change.